clifford/__init__.py

```diff
@@ -256,7 +256,7 @@ def grades_present(objin: 'MultiVector', threshold=0.0000001) -> Set[int]:
 # todo: work out how to let numba use the COO objects directly
 @_numba_utils.njit
 def _numba_val_get_left_gmt_matrix(x, k_list, l_list, m_list, mult_table_vals, ndims):
-    intermed = np.zeros((ndims, ndims))
+    intermed = np.zeros((ndims, ndims), dtype=x.dtype)
```

Comment:
Strictly this should be `result_type(x.dtype, mult_table_vals.dtype)`, but I think numba makes that awkward and we probably don't care.
```suggestion
    # TODO: consider `dtype=result_type(x.dtype, mult_table_vals.dtype)`
    intermed = np.zeros((ndims, ndims), dtype=x.dtype)
```